server/service/integration_mdm_test.go

```diff
@@ -34,7 +34,6 @@ import (
 	"github.com/fleetdm/fleet/v4/server/datastore/mysql"
 	"github.com/fleetdm/fleet/v4/server/datastore/redis/redistest"
 	"github.com/fleetdm/fleet/v4/server/fleet"
-	mdm_types "github.com/fleetdm/fleet/v4/server/fleet"
```

Comment:
Weird, we were importing package `fleet` twice?

Reply:
yes 😂 , I noticed we do this in a couple of places